ibm_db2/ibm_db2.php

```diff
@@ -1024,24 +1024,24 @@ function db2_specialcolumns() {}
  * </tr>
  * </table>
  */
-function db2_statistics($connection, $qualifier, $schema, $table_name, $unique) {}
+function db2_statistics($connection, ?string $qualifier, ?string $schema, string $table_name, bool|int $unique) {}
```

Comment:
Please update PHPDoc here as well

Reply:
Indeed. I must check this one in the extension. Thank you!
https://github.com/php/pecl-database-ibm_db2/blob/f4bfd692677db26cb16b5c3e72e0e61f122f05ac/ibm_db2.c#L3718

Reply:
Updated. See php/pecl-database-ibm_db2#50.